db.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+import json
+import sys
+
+
+def validate(input):
+    normalized = normalize_internal(input)
+    if normalized != input:
+        sys.stderr.write("Input is not normalized.\n")
+        sys.exit(0)
```

Comment:
Shouldn't this exit with an error, or CI will just pass with some stuff on stderr?